packages/student_id_alignment/README.md

```diff
@@ -0,0 +1,71 @@
+This bundle is meant to be used as a package with other bundles for the purpose of aligning student IDs from source data with known IDs from EdFi.
+
+## Usage
+
+To use this package:
+1. include this package in your earthmover project by adding:
+    ```yaml
+    packages:
+      student_id_alignment:
+        git: "https://github.com/edanalytics/earthmover_edfi_bundles.git"
+        branch: student_id_align
+        subdirectory: "packages/student_id_alignment"
+    ```
+1. provide a source of student IDs known to EdFi as either a query against an [Enable Data Union (EDU)](https://enabledataunion.org/) Ed-Fi data warehouse:
+    ```yaml
+    sources:
+      edfi_student_ids:
+        connection: "snowflake://$SNOWFLAKE_USER:$SNOWFLAKE_PASS@$SNOWFLAKE_ACCOUNT?warehouse=$SNOWFLAKE_WAREHOUSE"
+        # This is a Snowflake SQL query which constructs a structure like the student ID portion of
+        # Ed-Fi studentEducationOrganizationAssociations from EDU warehouse data:
+        query: >
+          select
```

Comment:
I think this could be constructed more consistently from `stg_ef3__stu_ed_org__identification_codes`

would likely want to filter on year as well

Reply:
sounds good, I updated the sample query accordingly